packages/flutter_adaptive_scaffold/example/lib/adaptive_scaffold_demo.dart

```diff
@@ -35,46 +42,64 @@ class MyHomePage extends StatelessWidget {
         Padding(
           padding: const EdgeInsets.all(8.0),
           child: Container(
-            color: const Color.fromARGB(255, 255, 201, 197),
+            color: const Color.fromARGB(255, 255, 29, 197),
```

Comment:
Why this color change?

Reply:
It was unintentional change. I can revert that.